contracts/Service.sol

```diff
@@ -1,4 +1,5 @@
-pragma solidity >=0.4.25 <0.7.0;
+// pragma solidity >=0.5.16 <0.9.0;
+pragma experimental ABIEncoderV2;
```

Comment:
The getFileInfo function return value is of type file (or something like that. I'm not 100% how data structures work in Solidity), which was formerly internal. I made it public since we need this on the frontend, but it appears returning a custom struct is only experimental for the moment.